api/internal/builtins/SortOrderTransformer.go

```diff
@@ -87,19 +87,10 @@ func (p *SortOrderTransformerPlugin) Transform(m resmap.ResMap) (err error) {
 // applyOrdering takes resources (given in ResMap) and a desired ordering given
 // as a sequence of ResIds, and updates the ResMap's resources to match the
 // ordering.
-func applyOrdering(m resmap.ResMap, ordering []resid.ResId) error {
```

Comment:
This file is generated code and should not be modified directly. 

Please update the corresponding file under `plugins` and use the `make` target that builds the plugins to generate this code.

Reply:
thanks, fixed! `make generate-kustomize-builtin-plugins`